### PR TITLE
update tracing

### PR DESCRIPTION
**Pull request by zirain**
fixes: https://github.com/istio/istio/issues/42838

1. update deprecated settings
2. add Telemetry API tips

### CODE REVIEW

**Review by kyessenov — APPROVED** [review]

File: content/en/docs/ops/integrations/jaeger/index.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ owner: istio/wg-environments-maintainers
 test: n/a
 ---
 
+{{< tip >}}
```
Comment (frankbu) [inline content/en/docs/ops/integrations/jaeger/index.md:10]: This tip should be move to boilerplate since it's used in three docs

Comment (zirain) [reply content/en/docs/ops/integrations/jaeger/index.md:10]: will do this in a following PR.

**Review by kfaseela — APPROVED** [review]

**Pull request merged**